app/javascript/packs/followButtons.js

```diff
@@ -51,23 +51,30 @@ function addButtonFollowText(button, style) {
  */
 function addAriaLabelToButton({ button, followType, followName, style = '' }) {
   let label = '';
+  let pressed = '';
   switch (style) {
     case 'follow':
       label = `Follow ${followType.toLowerCase()}: ${followName}`;
+      pressed = 'false';
       break;
     case 'follow-back':
       label = `Follow ${followType.toLowerCase()} back: ${followName}`;
+      pressed = 'true';
       break;
     case 'following':
       label = `Unfollow ${followType.toLowerCase()}: ${followName}`;
+      pressed = 'true';
       break;
     case 'self':
       label = `Edit profile`;
+      pressed = 'false';
```

Comment:
I don't think we want the 'Edit profile' button to be a toggle button (since it doesn't follow the same on/off state as the others).

Since the follow button helper automatically adds `aria-pressed="false"` (nice!), we'll need to remove the attribute here e.g. `button.removeAttribute('aria-pressed')`